activity/src/test/java/com/amazonaws/stepfunctions/cloudformation/activity/TranslatorTest.java

```diff
@@ -0,0 +1,39 @@
+package com.amazonaws.stepfunctions.cloudformation.activity;
+
+import com.amazonaws.services.stepfunctions.model.DescribeActivityResult;
+import com.amazonaws.services.stepfunctions.model.ListTagsForResourceResult;
+import com.amazonaws.services.stepfunctions.model.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TranslatorTest extends HandlerTestBase {
```

Comment:
curious: does this map to a class we have called `Translator`??  can't seem to find it

Reply:
Seems like it's copied/modified version of the similar thing being done for state machines, which has a different class name.

Reply:
I had originally named the class `Translator`, then updated it to `ResourceModelUtils` without updating the associated test class. I've updated the test class to be named `ResourceModelUtilsTest`, and have also moved the translation logic into a separate `Translator` class.